apiserver/resources/bob/conf.edn

```diff
@@ -12,8 +12,11 @@
              :api-url #or [#env BOB_QUEUE_API_URL "http://localhost:15672/api"]
              :conf {:exchanges #ref [:common :bob/queue-conf :exchanges]
                     :queues #ref [:common :bob/queue-conf :queues]}}
+ :bob/stream-env {:url #ref [:common :bob/stream-conf :url]
```

Comment:
this would be a problem i think, the url for queue has a port too and the stream would have a different port. worth having a dedicated conf for this i think.